src/components/Semester/OrderDropdown.vue

```diff
@@ -0,0 +1,154 @@
+<template>
```

Comment:
Maybe rename this file `ViewDropdown`

Reply:
<img width="1368" alt="Screen Shot 2021-10-20 at 4 51 56 PM" src="https://user-images.githubusercontent.com/73757337/138170299-99d56670-e230-403b-bc96-874bfa19a214.png">
I think the wrapping issue also occurs on master